Change one dialog of NPC on camp building in TALK_CAMP_TUTORIAL_MANSION json file

linzhongfeiniao opened the pull request:
#### Summary
None

#### Purpose of change

The previous dialog is too long and have 1 minor mistake on the counting of Kitchen (Count Kitchen 2 for twice). 
So the previous version is a little bit confusing to read. Shortened the text for clarity and easiness in maintaining.

#### Describe the solution

Shortened the text and listed the possible numbers of expansions for reader.

#### Describe alternatives you've considered

One possible alternative is changing the first Kitchen 2 to Kitchen 1 ONLY. But that would be too minor to propose one PR and not good for reading. 

#### Testing

Only changed text in dialog, won't introduce a bug.

#### Additional context

Found this one during localization. CDDA is great in many ways, one of that is the art of literature. Tidy instructions would greatly help that.

<!--README: Cataclysm: Dark Days Ahead is released under the Creative Commons Attribution ShareAlike 3.0 license.
The code and content of the game are free to use, modify, and redistribute for any purpose whatsoever.
By contributing to the project you agree to the terms of the license and that any contribution you make will also be covered by the same license.
See http://creativecommons.org/licenses/by-sa/3.0/ for details. -->

linzhongfeiniao committed:
Update dynamic_line for TALK_CAMP_TUTORIAL_MANSION

PatrikLundell commented:
You lost the point that the kitchens provide the largest sets of recipes. That doesn't really matter much any longer, as it's now possible to order companions to craft via dialog, not only via the bulletin board.

I-am-Erk commented on data/json/npcs/TALK_FACTION_CAMP.json:
```diff
@@ -276,7 +276,7 @@
     "id": "TALK_CAMP_TUTORIAL_MANSION",
     "type": "talk_topic",
-    "dynamic_line": "You can build a faction camp in a mansion.  You have four different starting positions available, namely the Fountain Garden, Dance Floor Room, Basketball Room, and Garden with Columns.  Depending on where you set up your base, the following site specific expansions may become available: Entrance 1, Entrance 2, Swimming Pool 1, Bedrooms 1, Unidentified Room, Kitchen 2, Librar ...
+    "dynamic_line": "You can build a faction camp in a mansion.  You have four different starting positions available, namely the Fountain Garden, Dance Floor Room, Basketball Room, and Garden with Columns.  Depending on where you set up your base, the following site specific expansions may become available: 2 Entrances, 2 Swimming Pools, 3 Bedrooms, 1 Unidentified Room, 2 Kitchens, 1 Library, 1  ...
```
```suggestion
    "dynamic_line": "You can build a faction camp in a mansion.  You have four different starting positions available, namely the Fountain Garden, Dance Floor Room, Basketball Room, and Garden with Columns.  Depending on where you set up your base, the following site specific expansions may become available: 2 Entrances, 2 Swimming Pools, 3 Bedrooms, 1 Unidentified Room, 2 Kitchens, 1 Library, 1 Bar and 1 Living Room.  The Entrances and Swimming Pool provide farming, the Library, Bar and Kitchens: a group of different sets of cooking recipes.",
```
there needs to be something linking these two clauses together

SchrodingerSig replied:
```suggestion
    "dynamic_line": "You can build a faction camp in a mansion. You have four different starting positions available, namely the Fountain Garden, Dance Floor Room, Basketball Room, and Garden with Columns. Depending on where you set up your base, the following site specific expansions may become available: Entrance 1&2, Swimming Pool 1&2, Bedrooms 1&2&3, Unidentified Room, Kitchen 1&2, Library, Bar, and Living Rooms. The Entrances and Swimming Pool 1 provide farming. Both the Library, Bar, and Kitchens extend the cooking recipe, with Kitchens having more options than Library, and Bar having it's own unique recipe set.",
```

*for refrence purpose, originally posted in zh-hans translation group chat and been asked by the author of this PR to post here.

linzhongfeiniao commented:
Ask for HELP:
I am restricted in my poor game experience that I just don't know if there is a real difference between the two swimming pools of mansion, since the original text seemed to separate the function for the two swimming pools: "The Entrances and Swimming Pool 1 provide farming."  So I just really don't know that. 
I think @SchrodingerSig 's translation edition is great. The only thing needs to check is the point above.

linzhongfeiniao committed:
Update data/json/npcs/TALK_FACTION_CAMP.json

Co-authored-by: Schrodinger_Sig <56752512+SchrodingerSig@users.noreply.github.com>

linzhongfeiniao committed:
Refactor dynamic_line for clarity and conciseness

more clarity in text. Got to check the original codes about what the real mechanisms are.

linzhongfeiniao committed:
Update TALK_FACTION_CAMP.json

Maleclypse committed:
Update data/json/npcs/TALK_FACTION_CAMP.json

Co-authored-by: Erk <45136638+I-am-Erk@users.noreply.github.com>

linzhongfeiniao committed:
Merge branch 'CleverRaven:master' into linzhongfeiniao-patch-1
